rusty_pizza_server/src/order_model/order.rs

```diff
@@ -0,0 +1,97 @@
+use crate::order_model::meals::Meals;
+use crate::order_model::user::User;
+use crate::util::money::Money;
+use std::collections::HashMap;
+use std::fmt::{self, Display, Formatter};
+use std::rc::Rc;
+
+#[derive(Debug, PartialEq)]
+enum OrderStatus {
+    Open,
+    Ordering,
+    Ordered(String),
+    Delivered,
+}
+
+impl fmt::Display for OrderStatus {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{:?}", self)
+    }
+}
```

Comment:
Was this added for a reason?

Reply:
Yes this is a kind of toString(). To print `Ordered(String)` in prettier way